### PR TITLE
feat(analytics): adding names registrations analytics

### PR DESCRIPTION
**Pull request by geekbrother**
# Description

This PR adds name registration analytics with the following data:
* Name,
* Owner's address,
* Registration chain ID (in case the user is not at the mainnet),
* Geo data: origin, region, country, continent.

The analytics data is stored in the `blockchain-api/name-requests/name_registrations` S3 bucket.

## How Has This Been Tested?

Not tested.

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/analytics/mod.rs [hunk 1]
```diff
@@ -287,6 +293,27 @@ impl RPCAnalytics {
...
+
+        let observer = Observer(DataKind::NameRegistrations);
+        let name_registrations = BatchCollector::new(
+            CollectorConfig {
+                data_queue_capacity: DATA_QUEUE_CAPACITY,
+                ..Default::default()
+            },
+            ParquetBatchFactory::new(Default::default()).with_observer(observer),
+            AwsExporter::new(AwsConfig {
+                export_prefix: "blockchain-api/name-requests".to_owned(),
```
Comment (chris13524) [inline src/analytics/mod.rs:312]:
```suggestion
                export_prefix: "blockchain-api/name-registrations".to_owned(),
```

**Review by chris13524 — APPROVED** [review]
Is registration all we want to track? What about name updates? Should this be modeled in a different way to support?

**Review by nopestack — APPROVED** [review]

**Comment by geekbrother** [comment]
> Is registration all we want to track? What about name updates?

The registration is the only we need at the moment since we do not have UI for the name updates yet (( Just the registration and resolution.

>  Should this be modeled in a different way to support?

It should be a different struct for updates.

**Review by dnul — APPROVED** [review]
LGTM! sorry for the delay!

**Pull request merged**